src/Diagnostics.ModelsAndUtils/Models/GeoMaster/VnetParameters.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Diagnostics.ModelsAndUtils.Models
+{
+    public class VnetParameters
```

Comment:
I see this is an input model is used only by geomaster data provider
Is this also getting used in csx scripts?

Reply:
No. GeoMasterDataProvider just takes exact string parameters for subscriptionid, vnet subnet name and etc,
@puneetg1983 , you should probably change the access modifier for this model to internal so as to not expose this to developers creating their csx file.

Reply:
This one I can make internal.